acceptance-tests/src/test/js/multibranch/multibranchOpening.js

```diff
@@ -98,8 +98,9 @@ module.exports = {
          blueActivityPage.waitForElementVisible('.branches');
          blueActivityPage.click(".branches");
 
-         blueActivityPage.waitForElementVisible('tr[id^="feature"]');
-         blueActivityPage.click('tr[id^="feature"]');
+         const rowSelector = '.JTable-row[data-branch="feature"]'
```

Comment:
:bug: needs to be feature/1 - as that is the data value (before it was a partial match).